src/test/java/hudson/maven/Maven31xBuildTest.java

```diff
@@ -49,6 +34,9 @@
     public MavenInstallation configureMaven3x()
         throws Exception
     {
-        return configureMaven31();
+        MavenInstallation mvn = ToolInstallations.configureDefaultMaven("apache-maven-3.1.0", MavenInstallation.MAVEN_30);
+        MavenInstallation m3 = new MavenInstallation("apache-maven-3.1.0", mvn.getHome(), NO_PROPERTIES);
+        jenkins.getDescriptorByType(Maven.DescriptorImpl.class).setInstallations(m3);
```

Comment:
`ToolInstallations` did not get `configureMaven31`. Probably only used in this plugin anyway.